lib/ddtrace/contrib/rack/middlewares.rb

```diff
@@ -104,7 +105,8 @@ def set_request_tags!(request_span, env, status, headers, response, original_env
             request_span.set_tag(Datadog::Ext::HTTP::METHOD, env['REQUEST_METHOD'])
           end
           if request_span.get_tag(Datadog::Ext::HTTP::URL).nil?
-            request_span.set_tag(Datadog::Ext::HTTP::URL, url)
+            options = Datadog.configuration[:rack][:quantize]
+            request_span.set_tag(Datadog::Ext::HTTP::URL, Quantize.format_url(url, options))
```

Comment:
can we handle any kind of exception here? since it's the first middleware, in case of an issue (i.e. encoding/decoding/filtering problem) would be great to avoid propagating our internal exception. I think it's enough to handle the exception somewhere, and log it as `debug()`.

Reply:
@palazzem are you talking about this tagging function in general? Or specifically the use of quantization here?